packages/purpose-process/test/deletePurposeVersion.test.ts

```diff
@@ -249,4 +457,177 @@ describe("deletePurposeVersion", () => {
       purposeVersionCannotBeDeleted(mockPurpose.id, mockPurposeVersion.id)
     );
   });
+  it("should throw organizationIsNotTheDelegatedConsumer when the requester is the Consumer and is deleting a purpose version created by the delegate in deletePurposeVersion", async () => {
+    const authData = getRandomAuthData();
+    const mockPurposeVersion: PurposeVersion = {
+      ...getMockPurposeVersion(),
+      state: purposeVersionState.active,
```

Comment:
```suggestion
      state: randomArrayitem(deletableStates),
```
`active` isn't a valid deletable state